GroceryMarket/ViewControllers/HomeViewController/HomeViewController.m

```diff
@@ -33,6 +34,13 @@ - (void)viewWillAppear:(BOOL)animated {
     [profileVC.tabBarItem setTitle:@"Account"];
     [mapVC.tabBarItem setTitle:@"Map"];
     
+//    self.tabBar.layer.masksToBounds = true;
+//    self.tabBar.layer.cornerRadius = 30;
+//    self.tabBar.layer.maskedCorners = kCALayerMinXMinYCorner | kCALayerMaxXMinYCorner;
+    // [.layerMinXMinYCorner, .layerMaxXMinYCorner]
```

Comment:
nit: try not to merge any commented code into master, unless they are used as comments (for others to understand the code).